### PR TITLE
netns: add documentation on how to go back on default netns

### PR DESCRIPTION
**Pull request by ffourcot**
Hello,

I didn't find any reference in the documentation for this annoying thing. There is perhaps a better way, but in all cases I think that it's a good idea to add documentation on it.

### CODE REVIEW

File: pyroute2/netns/__init__.py [hunk 1]
```diff
@@ -222,6 +222,11 @@ def setns(netns, flags=os.O_CREAT, libc=None):
         - O_CREAT -- create netns, if doesn't exist
         - O_CREAT | O_EXCL -- create only if doesn't exist
 
+    Note that "main" netns as no name. But you can access it with::
```
Comment (svinota) [inline pyroute2/netns/__init__.py]: … has no name

**Comment by svinota** [comment]
Another way is to open `/proc/self/ns/net`, save the fd, and switch back with `setns()`. This allows to revert to own namespace even if it was not the main one.

**Comment by svinota** [comment]
Probably I have to dump it also to the docstring.

**Comment by svinota** [comment]
As an idea, looks at the commit — e7810d93cc330e35cd3fba38952b683cf0e0994b

**Comment by ffourcot** [comment]
Thank you!

**Comment by svinota** [comment]
Thank you for the issue. I had to implement it a long time ago, but didn't.

**Pull request merged**